### PR TITLE
Move internal examples to `integration_tests` crate

### PR DESCRIPTION
**Pull request by brzep**
issue #603 - `examples` dir refactor + cleanup

moved examples to integration_tests, moved common.rs to examples_common.rs in integration_tests/src + needed modifications

minor bugs corrections

### CODE REVIEW

File: integration_tests/Cargo.toml [hunk 1]
```diff
@@ -26,3 +33,9 @@ tracing = { workspace = true }
 opus = { workspace = true }
 pitch-detection = "0.3.0"
 rand = { workspace = true }
+futures-util = { workspace = true }
+log = { workspace = true }
+tokio = { workspace = true }
+signal-hook = { workspace = true }
+compositor_chromium = { path = "../compositor_chromium", optional = true }
+tokio-tungstenite = "0.21.0"
```
Comment (wkozyra95) [inline integration_tests/Cargo.toml:41]: fix newline, (I recommend configuring that in vscode)

File: integration_tests/src/examples_common.rs
Comment (wkozyra95) [inline integration_tests/src/examples_common.rs:1]: Maybe `examples_common` -> `examples`? 

Also, it would be good to split that code in a follow-up PR.

**Review by wkozyra95 — APPROVED** [review]

**Comment by wkozyra95** [comment]
btw renamed the PR to be more inlined with the style we use for commits

**Review by WojciechBarczynski — APPROVED** [review]

**Comment by WojciechBarczynski** [comment]
Don't link this PR as closing #603 
This is just the first part. You can link it in the description in such cases (just use #issue_no).
If you link it that way, it'll close after the merge.

**Pull request merged**